lib/utils/fs.go

```diff
@@ -209,3 +211,42 @@ func FSTryReadLockTimeout(ctx context.Context, filePath string, timeout time.Dur
 
 	return unlockWrapper(fileLock.Unlock, fileLock.Path()), nil
 }
+
+// RemoveSecure attempts to securely delete the file by first overwriting the file with random data followed by
+// calling os.Remove(filePath). The number of times to overwrite the data is set by iterations. Iterations must
+// be > 0 or a trace.BadParameterError will be returned.
+func RemoveSecure(filePath string, iterations int) error {
```

Comment:
WDYT about picking a good default for the number of iterations and removing the param?

Reply:
I agree w/ this - if there's a standard for us to abide by I don't see us ever needing to customise this !

Reply:
This seems reasonable to me. I made it configurable because that's what most security software does but if we have a need for this in the future we can always add another method.